disdrodb/utils/zip.py

```diff
@@ -13,3 +14,16 @@ def _unzip_file(file_path: str, dest_path: str) -> None:
 
     with zipfile.ZipFile(file_path, "r") as zip_ref:
         zip_ref.extractall(dest_path)
+
+
+def _zip_dir(dir_path: str) -> str:

```

Comment:
Can you add in this file also an utility function that compress each single file within a station directory? 
Within the station directory there might be other directory (like a tree /year/month) so the function should just compress files ! 
Ideally would be nice to allow various compession option like: `gzip, zip, bzip`.

Reply:
done